### PR TITLE
Added args to Jira-edit-issue command

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35216

## Description
Added components security environment arguments to jira-edit-issue command.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 1]
```diff
@@ -652,15 +695,13 @@ script:
...
+      for debugging purposes.
     execution: false
     name: get-modified-remote-data
-  dockerimage: demisto/oauthlib:1.0.0.19245
+  dockerimage: demisto/oauthlib:1.0.0.20579
   feed: false
   isfetch: true
-  ismappable: true
-  isremotesyncin: true
-  isremotesyncout: true
```
Comment (yuvalbenshalom) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:663]: why this was removed? formatting?

**Review by yuvalbenshalom — COMMENTED** [review]
links good, see my question.

**Review by guykeller — COMMENTED** [review]
Other than what Yuval asked, looks good to me.

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml (hunk 1 above)
Comment (ChanochShayner) [reply Packs/Jira/Integrations/JiraV2/JiraV2.yml]: fixed

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/Jira/ReleaseNotes/1_3_4.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Added **components** **security** **environment** arguments to ***jira-edit-issue*** and ***ira-create-issue*** commands.
```
Comment (amshamah419) [inline Packs/Jira/ReleaseNotes/1_3_4.md:4]:
```suggestion
- Added the **components**, **security**, and **environment** arguments to the ***jira-edit-issue*** and ***ira-create-issue*** commands.
```

**Pull request merged**